deployments/cloudfoundry/bosh/jobs/splunk-otel-collector/templates/otel-collector-config.yaml.erb

```diff
@@ -23,8 +23,11 @@ receivers:
         <% end %>
 
 processors:
-  resourcedetection:
```

Comment:
Can we somehow test this file so it doesn't get stale like it did here?

Reply:
There isn't a quick way to test this automatically. Running a BOSH deployment requires a lot of set up and dependency installs, so it'd be some amount of effort. This file doesn't get changed much and is tested directly with each Tanzu Tile release.

The change to the config here is a bug fix, it didn't actually go stale.